content/v2.0/query-data/optimize-queries.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Optimize Flux queries
+description: >
+  Optimize your Flux queries by changing function order, understanding how functions work,
+  and being conscious of the nature of the data queried.
+weight: 104
+menu:
+  v2_0:
+    name: Optimize queries
+    parent: Query data
+v2.0/tags: [query]
+---
+
+Optimize your Flux queries with a few simple principles.
+Query optimizations center around reducing the memory and compute (CPU) requirements
+of a query by changing function order, understanding how functions work, and being
+conscious of the nature of the data queried.
+
+## Start queries with pushdown functions
+Certain Flux functions can push their data manipulation down to the underlying
+data source rather than pulling the data into memory and manipulating it there.
+Using "pushdown" functions reduces the amount of memory necessary to run a query.
+However, to benefit from these performance gains, you must **use pushdown functions
+at the beginning of your query**.
+When a non-pushdown function runs, Flux pulls the data into memory and manipulates the data there.
+All subsequent functions must operate in memory, including pushdown-capable functions.
```

Comment:
Most? Flux functions manipulate data in memory. Flux pushdown functions reduce the memory required, but require data manipulation in an underlying data source? To increase query performance, you must **use pushdown functions
at the beginning of your query**.